[2024-05-29T17:03:02.397+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-29T17:03:02.671+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: test_1.task_2 scheduled__2024-03-10T00:00:00+00:00 [queued]>
[2024-05-29T17:03:02.818+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: test_1.task_2 scheduled__2024-03-10T00:00:00+00:00 [queued]>
[2024-05-29T17:03:02.832+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-05-29T17:03:02.983+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): task_2> on 2024-03-10 00:00:00+00:00
[2024-05-29T17:03:03.174+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=317) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-29T17:03:03.266+0000] {standard_task_runner.py:63} INFO - Started process 356 to run task
[2024-05-29T17:03:03.201+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'test_1', 'task_2', 'scheduled__2024-03-10T00:00:00+00:00', '--job-id', '55', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmp2v975ffm']
[2024-05-29T17:03:03.285+0000] {standard_task_runner.py:91} INFO - Job 55: Subtask task_2
[2024-05-29T17:03:03.815+0000] {task_command.py:426} INFO - Running <TaskInstance: test_1.task_2 scheduled__2024-03-10T00:00:00+00:00 [running]> on host 2832635d2b45
[2024-05-29T17:03:05.198+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='test_1' AIRFLOW_CTX_TASK_ID='task_2' AIRFLOW_CTX_EXECUTION_DATE='2024-03-10T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-03-10T00:00:00+00:00'
[2024-05-29T17:03:05.246+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-29T17:03:05.251+0000] {logging_mixin.py:188} INFO - Run function_2!
[2024-05-29T17:03:05.262+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-29T17:03:05.305+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-29T17:03:05.373+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=test_1, task_id=task_2, run_id=scheduled__2024-03-10T00:00:00+00:00, execution_date=20240310T000000, start_date=20240529T170302, end_date=20240529T170305
[2024-05-29T17:03:05.567+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-29T17:03:05.698+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-29T17:03:05.730+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
